metricbeat/module/vsphere/datastore/_meta/data.json

```diff
@@ -10,39 +10,51 @@
         "period": 10000
     },
     "service": {
-        "address": "127.0.0.1:55632",
+        "address": "127.0.0.1:33365",
         "type": "vsphere"
     },
     "vsphere": {
         "datastore": {
+            "host": {
+                "count": 1,
+                "names": ["DC3_H0"]
+            },
+            "status": "green",
+            "vm": {
+                "count": 6,
+                "names": ["DC3_H0_VM0"]
+            },
+            "read": {
+                "bytes": 0
+            },
+            "write": {
+                "bytes": 337000
+            },
+            "disk": {
+                "capacity": {
+                    "usage": {
+                        "bytes": 520505786368
+                    },
+                    "bytes": 1610344300544
+                },
+                "provisioned": {
```

Comment:
I assumed we were not getting these metrics. Are these dummy values or generated through simulator ?

Reply:
These values are from actual results we got from our instance.